config/config.go

```diff
@@ -392,6 +392,33 @@ type RPCConfig struct {
 	// predictability in subscription behavior.
 	CloseOnSlowClient bool `mapstructure:"experimental_close_on_slow_client"`
 
+	// If true, disable the websocket interface to the RPC service.  This has
+	// the effect of disabling the /subscribe, /unsubscribe, and /unsubscribe_all
+	// methods for event subscription.
+	//
+	// EXPERIMENTAL: This setting will be removed in Tendermint v0.37.
```

Comment:
This comment might need to be updated, since I believe v0.37 is the next scheduled release.